src/components/ec-input-field/ec-input-field.vue

```diff
@@ -206,6 +206,9 @@ export default {
     focus() {
       this.$refs.input.focus();
     },
+    getInputRef() {
```

Comment:
This is not a leftover, we use it above in the datepicker to mount in a unique reference the flatpickr

Reply:
@jafcalvente Here we use this unique ref. We can solve the issue of not opening the calendar when clicking on the icon by instead targetting the input to target the input wrapper. This will make it open when we click on the icon as well.